package.json

```diff
@@ -71,7 +71,6 @@
   "scripts": {
     "clean": "rimraf logs/*",
     "lint": "jscs .",
-    "pretest": "npm run clean && npm run lint",
```

Comment:
(it was a joke - I re-instated it. My original PR was on top of an outdated branch and forget to bring it back in).